### DISCUSSION TITLE
Config option for bonemealable Sugar Cane

### DISCUSSION THREAD

**Discussion by DuneSciFye** [PurpurMC/Purpur#1096] [__Completed__] [answered]
An option to make sugar cane bone mealable

**Comment by Tydium** [comment]
😭

**Comment by granny** [comment, marked as answer]
Implemented as of build #1911, commit https://github.com/PurpurMC/Purpur/commit/bac7738314934b25d969bcfd350ef57213b2a571.

There's a chance it will be moved to [PurpurExtra](https://github.com/PurpurMC/PurpurExtras) in the future.
